src/shared-components/utils/Flex/Flex.module.css

```diff
@@ -1,12 +1,11 @@
 /*
-Copyright 2024 New Vector Ltd.
-Copyright 2023 The Matrix.org Foundation C.I.C.
+ * Copyright 2025 New Vector Ltd.
```

Comment:
I don't think this can just shed the old copyrights given the majority of the file hasn't changed

Reply:
Done in https://github.com/element-hq/element-web/pull/30357/commits/7b4f240390aede76cd4b055b25426d49cc82f360